app/actions.ts

```diff
@@ -107,6 +112,34 @@ export async function clearChats() {
   return redirect('/')
 }
 
+export async function deleteSavedFiles(chatId: string, userId: string, path?: string) {
+  const session = await auth()
+
+  if (session?.user?.id !== userId) {
+    return {
+      error: 'Unauthorized'
+    }
+  }
+  const chat = await getChat(chatId, userId) as Chat
+  if (!chat || 'error' in chat) {
```

Comment:
same for 'error' https://github.com/Conrad-Labs/nextjs-chatbot-ui-template/pull/6/files#r1879844154

Reply:
done